### PR TITLE
Basic ES6 conversions: let/const and arrow functions

### PR DESCRIPTION
**Pull request by camillobruni**
This is the first, format and  function preserving PR to start the ES6.

- Convert var to let/const
- Use arrow functions where possible

### CODE REVIEW

File: InteractiveRunner.html [hunk 1]
```diff
@@ -115,26 +115,26 @@
...
-                for (var testName in suiteResults.tests) {
-                    var testResults = suiteResults.tests[testName];
-                    for (var subtestName in testResults.tests) {
+        didRunSuites(measuredValues) {
+            let results = '';
+            for (let suiteName in measuredValues.tests) {
+                let suiteResults = measuredValues.tests[suiteName];
+                for (let testName in suiteResults.tests) {
+                    let testResults = suiteResults.tests[testName];
+                    for (let subtestName in testResults.tests) {
```
Comment (rniwa) [inline InteractiveRunner.html:137]: We can use const in these instances.

File: resources/benchmark-report.js [hunk 2]
```diff
@@ -38,29 +38,29 @@
...
-                for (var suiteName in measuredValues.tests) {
-                    var suite = measuredValues.tests[suiteName];
-                    for (var testName in suite.tests) {
-                        var test = suite.tests[testName];
-                        for (var subtestName in test.tests)
+                for (let suiteName in measuredValues.tests) {
+                    const suite = measuredValues.tests[suiteName];
+                    for (let testName in suite.tests) {
+                        const test = suite.tests[testName];
+                        for (let subtestName in test.tests)
```
Comment (rniwa) [inline resources/benchmark-report.js:63]: Ditto about using const in for loops.

File: resources/benchmark-report.js [hunk 3]
```diff
@@ -70,12 +70,12 @@
 
             PerfTestRunner.reportValues(createTest(null, null, false, 'pt'), scores);
 
-            var fullNames = new Array;
-            for (var fullName in measuredValuesByFullName)
+            const fullNames = new Array;
+            for (let fullName in measuredValuesByFullName)
```
Comment (rniwa) [inline resources/benchmark-report.js:74]: Use const

File: resources/benchmark-runner.js [hunk 4]
```diff
@@ -222,19 +222,19 @@ BenchmarkRunner.prototype._finalize = function () {
     this._removeFrame();
 
     if (this._client && this._client.didRunSuites) {
-        var product = 1;
-        var values = [];
-        for (var suiteName in this._measuredValues.tests) {
-            var suiteTotal = this._measuredValues.tests[suiteName].total;
+        let product = 1;
+        const values = [];
+        for (let suiteName in this._measuredValues.tests) {
```
Comment (rniwa) [inline resources/benchmark-runner.js:227]: Use const.

**Comment by bgrins** [comment]
This type of automated migration is fine with me - if rniwa approves, feel free to land away

File: InteractiveRunner.html (hunk 1 above)
Comment (camillobruni) [reply InteractiveRunner.html]: Ah good point, I always forget about the for-in/for-of variant. 👍 

DONE

**Pull request merged**